lib/cocoaseeds.rb

```diff
@@ -3,6 +3,7 @@
 require 'fileutils'
 require 'xcodeproj'
 require 'yaml'
+require 'shellwords'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.